doc/plugins.dox

```diff
@@ -329,24 +422,30 @@ Now we need to clean up properly when the plugin is unloaded.
 To remove the menu item from the Tools menu, you can use gtk_widget_destroy().
 gtk_widget_destroy() expects a pointer to a GtkWidget object.
 
-First you should add gtk_widget_destroy() to your plugin_cleanup() function.
-The argument for gtk_widget_destroy() is the widget object you created earlier in
-plugin_init(). To be able to access this pointer in plugin_cleanup(), you need to move
-its definition from plugin_init() into the global context so its visibility will increase
+First you should add gtk_widget_destroy() to your @ref GeanyPluginFuncs::cleanup() function. The
```

Comment:
I refer not to @c all kinds of words even if they're function names. It makes the HTML output harder to read. Documented functions are already automatically detected and links are created (but with the normal font).